packages/docs/src/components/layout/VersionSwitcher.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ChangeEvent } from 'react';
+import corePackage from '../../../../design-system/package.json';
+import healthcarePackage from '../../../../ds-healthcare-gov/package.json';
+import medicarePackage from '../../../../ds-medicare-gov/package.json';
+import themes from '../../../../../themes.json';
+import versions from '../../../../../versions.json';
+import useTheme from '../../helpers/useTheme';
+import { Dropdown } from '@cmsgov/design-system';
+
+function getVersionOptions(versions: string[]) {
+  return versions.map((version) => ({
+    label: version,
+    value: version,
+  }));
+}
+
+function getPackageData(theme: string) {
+  switch (theme) {
+    case 'core':
+      return corePackage;
+    case 'healthcare':
+      return healthcarePackage;
+    case 'medicare':
+      return medicarePackage;
+  }
+  return { version: '' };
+}
+
+/**
+ * Theme Switcher
+ * The dropdown component to switch the theme of the documentation site
+ */
+const ThemeSwitcher = () => {
```

Comment:
We have another ThemeSwitcher component. Should this also be called ThemeSwitcher? That and the above comment make me a little confused on what the difference between the components is.

Reply:
Doh! Good catch

Reply:
Copypasta 🍝